### ISSUE TITLE
parse environment

### ISSUE THREAD

**Issue by digitalheir** [digitalheir/latex-parser#1]
sort of works already?

add tests for

- named environment filled with content
- anonymous environment

**Comment by PythonNut** [comment]
I don't think this quite works in the case where the environment name contains a `*`.
